controllers/mdutil/util.go

```diff
@@ -42,27 +42,6 @@ const (
 	// DefaultMachineDeploymentUniqueLabelKey is the label applied to Machines
 	// in a MachineDeployment containing the hash of the template.
 	DefaultMachineDeploymentUniqueLabelKey = "machine-template-hash"
```

Comment:
Note/fyi This whole package will be deprecated and removed in a follow-up PR to https://github.com/kubernetes-sigs/cluster-api/pull/5250/files

(but your change here is fine)